FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -192,14 +208,15 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 while several stories are omitted and the numbering being confused: no. 13 omitted, no. 42 omitted, 
                                 one miracle after no. 89 and before no. 90 has no number, no. 100 and no. 101 omitted, no 121 is omitted.</note>
                             <note>Some stories on <locus target="93"></locus> are similar to the some passages in the Life of Cyrus as it is recorded in Ms. BMorient701 (= Wright 277).</note>
+                                <colophon xml:id="coloph1"><locus target="113vb" xml:lang="gez"/>፻፴፬፡ ዘተጽሕፈ፡ ተአምር፡ ዘማርያም። በእዴሁ፡ ለ<persName ref="PRS0000">አርሳንዮስ፡</persName> ሊቀ፡ ካህናት፡</colophon>
                         </msItem>
-                        <colophon xml:id="coloph1"><locus target="113vb" xml:lang="gez">፻፴፬፡ ዘተጽሕፈ፡ ተአምር፡ ዘማርያም። በእዴሁ፡ ለ<persName>አርሳንዮስ፡</persName> ሊቀ፡ ካህናት፡</locus></colophon>
+                        
                         <msItem xml:id="p1_i2">
                             <locus from="113vb" to="122rb"></locus>
                             <locus from="123ra" to="127va"></locus>
-                            <layout columns="2" writtenLines="35"></layout>
```

Comment:
layout belongs to codicology

Reply:
All these things have already been corrected, please update your local version by clicking PULL in your GitHub Desktop